python/python/lance/ray/sink.py

```diff
@@ -0,0 +1,133 @@
+# SPDX-License-Identifier: Apache-2.0
+# SPDX-FileCopyrightText: Copyright The Lance Authors
+
+import logging
+from typing import TYPE_CHECKING, Iterable, Optional, Union
+
+import pyarrow as pa
+
+import lance
+from lance.fragment import FragmentMetadata, LanceFragment
+
+from ..dependencies import ray
+
+if TYPE_CHECKING:
+    import pandas as pd
+
+__all__ = ["LanceDatasink"]
+
+
+class LanceDatasink(ray.data.Datasink):
+    """Lance Ray Datasink.
+
+    Write a Ray dataset to lance.
+
+    Parameters
+    ----------
+    uri : str
+        The base URI of the dataset.
+    schema : pyarrow.Schema
+        The schema of the dataset.
+    mode : str, optional
+        The write mode. Default is 'append'. Choices are 'append' and 'overwrite'.
```

Comment:
Looks like the default is create which isn't even mentioned here?

Reply:
Fixed.